src/test/java/seedu/uninurse/testutil/TypicalIndexes.java

```diff
@@ -12,4 +12,7 @@ public class TypicalIndexes {
     public static final Index INDEX_FIRST_TASK = Index.fromOneBased(1);
     public static final Index INDEX_SECOND_TASK = Index.fromOneBased(2);
     public static final Index INDEX_THIRD_TASK = Index.fromOneBased(3);
+    public static final Index INDEX_FIRST_ATTRIBUTE = Index.fromOneBased(1);
+    public static final Index INDEX_SECOND_ATTRIBUTE = Index.fromOneBased(2);
+    public static final Index INDEX_THIRD_ATTRIBUTE = Index.fromOneBased(3);
```

Comment:
Oh I left it there as the merge from your other PR should delete the repeated instances of `INDEX_XX_TASK` as I did not want to modify the other classes since it is already handled in your PR.